fgrpc/grpcrunner.go

```diff
@@ -156,9 +160,15 @@ func RunGRPCTest(o *GRPCRunnerOptions) (*GRPCRunnerResults, error) {
 	return &total, nil
 }
 
-// GRPCDestination parses dest and returns dest:port based on dest type
+// grpcDestination parses dest and returns dest:port based on dest type
 // being a hostname, IP address or hostname/ip:port pair.
-func GRPCDestination(dest string) string {
+func grpcDestination(dest string) string {
+	// strip any unintentional http/https scheme prefixes from destination
+	if strings.HasPrefix(dest, "http://") || strings.HasPrefix(dest, "https://") {
+		dest = strings.Replace(strings.Replace(dest, "https://", "", 1), "http://", "", 1)
+		log.Warnf("stripping http/https scheme. grpc destination: %v", dest)
```

Comment:
it's a feature so probably Infof or LogVf

Reply:
changed to infof